rtsf-at-checkout-device-scale/Dockerfile

```diff
@@ -19,16 +19,14 @@ RUN go mod download
 COPY . .
 
 RUN go mod tidy
-RUN CGO_ENABLED=1 go build -o device-scale cmd/main.go
+RUN go build -tags no_zmq -o device-scale cmd/main.go
```

Comment:
Note: In Minnesota (current Edgex WIP) this build tag has been removed with the removal of ZMQ.  So for the Minnesota EdgeX services/sdks this is not needed.